src/Aquifer.API/Endpoints/TranslationPairs/Patch/Validator.cs

```diff
@@ -8,6 +8,6 @@ public class Validator : Validator<Request>
     public Validator()
     {
         RuleFor(x => x.Key).MinimumLength(3).NotEmpty().When(x => x.Value is null);
-        RuleFor(x => x.Value).MinimumLength(3).NotEmpty().When(x => x.Key is null);
+        RuleFor(x => x.Value).MinimumLength(2).NotEmpty().When(x => x.Key is null);
```

Comment:
It looks like the CMS also allows 1 character: https://biblionexusworkspace.slack.com/archives/C0662HZ7RKM/p1736875550715359